packages/components/components/elvis-datepicker/src/react/calendar/calendar.tsx

```diff
@@ -156,12 +158,27 @@ export const Calendar: React.FC<Props> = ({
     }
   }, [hoveredDate]);
 
+  const labels =
+    lang === 'no'
+      ? {
+          prevMonth: 'Forrige måned',
+          nextMonth: 'Neste måned',
+          calendarDescription: 'Kalender. Bruk piltaster for navigasjon.',
+          resetDate: 'Nullstill dato',
+        }
+      : {
+          prevMonth: 'Previous month',
+          nextMonth: 'Next month',
+          calendarDescription: 'Calendar. Use arrow keys for navigation.',
+          resetDate: 'Reset date',
+        };
+
   return (
     <Container>
       <CalendarHeader>
         <IconButton
           onClick={() => shuffleViewedMonth(-1)}
-          aria-label="Forrige måned"
+          aria-label={`${labels.prevMonth}`}
```

Comment:
You probably don't need the template string here as `labels.prevMonth`, `labels.nextMonth`, `labels.calendarDescription` and`labels.resetDate` all are strings